core/src/test/java/nl/nn/adapterframework/jdbc/FixedQuerySenderTest.java

```diff
@@ -7,38 +7,64 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assume.assumeThat;
-import static org.junit.Assume.assumeTrue;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
 import java.io.ByteArrayInputStream;
+import java.io.Serializable;
 import java.io.StringReader;
 
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
 
+import nl.nn.adapterframework.core.PipeLineSession;
 import nl.nn.adapterframework.core.SenderException;
 import nl.nn.adapterframework.functional.ThrowingConsumer;
 import nl.nn.adapterframework.parameters.Parameter;
-import nl.nn.adapterframework.parameters.Parameter.ParameterType;
 import nl.nn.adapterframework.stream.Message;
 import nl.nn.adapterframework.stream.document.DocumentFormat;
 import nl.nn.adapterframework.testutil.ParameterBuilder;
+import nl.nn.adapterframework.testutil.TestConfiguration;
 import nl.nn.adapterframework.testutil.TestFileUtils;
+import nl.nn.adapterframework.testutil.TransactionManagerType;
+import nl.nn.adapterframework.testutil.junit.DatabaseTest;
+import nl.nn.adapterframework.testutil.junit.DatabaseTestEnvironment;
+import nl.nn.adapterframework.testutil.junit.WithLiquibase;
 
-public class FixedQuerySenderTest extends JdbcSenderTestBase<FixedQuerySender> {
+@WithLiquibase(tableName = FixedQuerySenderTest.tableName, file = "Migrator/JdbcTestBaseQuery.xml")
+public class FixedQuerySenderTest {
 
-	private final String resultColumnsReturned = "<result><rowset><row number=\"0\"><field name=\"TKEY\">1</field><field name=\"TVARCHAR\">value</field></row></rowset></result>";
+	private FixedQuerySender fixedQuerySender;
+	protected static final String tableName = "FQS_TABLE";
```

Comment:
```suggestion
	protected static final String TABLE_NAME = "FQS_TABLE";
```
Effe lokaal refactoren, zodat je elke instance meeneemt.